build.sh

```diff
@@ -18,7 +18,7 @@ echo "getting the file from gnu.org mirror"
 if [ ! -e ${SRC_DIR}/${SOURCE_FILE}.lock ] && [ ! -s ${SRC_DIR}/${SOURCE_FILE} ] ; then
 # claim the download
   touch  ${SRC_DIR}/${SOURCE_FILE}.lock
-#TODO @mtorrisi   wget -O ${SRC_DIR}/${SOURCE_FILE}
+  wget --no-cookies --no-check-certificate --header "Cookie: gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie" "http://download.oracle.com/otn-pub/java/jdk/8u66-b17/jdk-8u66-linux-x64.tar.gz" -O ${SRC_DIR}/${SOURCE_FILE}
```

Comment:
you need to call the file by version, since there are two versions of the tarball. 

You see the "VERSION" bit : 
# VERSION is set to
# 8u66 7u51 

this variable needs to be used in defining the url.

We also need to figure out how to include the url path in the variable. For this example of java-8, the path `8u66-b17/jdk-8u66-linux-x64.tar.gz` can be translated as : 

`${VERSION}-b17/${SOURCE_FILE}` - so we need a way of encoding the "b17" into the version bit.

We could add a variable to the environment to define the "build number", or we could do a recursive download by ignoring `${SOURCE_FILE}` and just defining the version as `8u66-b17`.